statement.md

```diff
@@ -0,0 +1,84 @@
+
+JupyterLab is not compatible with:
+
+* Internet Explorer, Edge; JAWS, NVDA, VoiceOver, Narrator, Orca screen readers; voice control technology.
+
+### Technical specifications
+
+Accessibility of JupyterLab relies on the following technologies to work with the particular combination of web browser and any assistive technologies or plugins installed on your computer:
+
+* HTML
+* WAI-ARIA
+* CSS
+* JavaScript
+
+These technologies are relied upon for conformance with the accessibility standards used.
+
+### Limitations and alternatives
+
+Despite our best efforts to ensure accessibility of JupyterLab, there may be some limitations. Below is a description of known limitations, and potential solutions. Please contact us if you observe an issue not listed below.
+
+Known limitations for JupyterLab:
+
+1. **Documents**: Documents writen by the community may not include accessible content because we do not and cannot review every document that can be opened in JupyterLab. To support accessible documents, we are drafting guidelines for accessible document content with an emphasis on Jupyter notebooks. Please report the issue to the author and [open an issue on jupyter/accessibility](https://github.com/jupyter/accessibility/issues/new) describing the problem and the behavior you expect so we may integrate it into our content guidelines.
+    
+2. **JupyterLab extensions**: JupyterLab extensions writen by the community may not be accessible because JupyterLab extensions can be written by anyone in the community and have no required review process. We do not and can not review every JupyterLab extension. To support accessible extensions, we encourage extension authors to use existing, accessible JupyterLab components for their extension. We also  provide periodic opportunities for community education on accessibility. Please report the issue to the author and let them know the [jupyter/accessibility](https://github.com/jupyter/accessibility/) community may be able to provide guidance.
+
+### Assessment approach
+
+Jupyter accessibility contributors assessed the accessibility of JupyterLab by the following approaches:
+
+* Self-evaluation
+* Regular automated testing to monitor for regressions (can be found at [jupyter/accessibility](https://github.com/jupyter/accessibility)
+* User feedback</li>
+
+### Evaluation report
+
+An evaluation for JupyterLab is available at: [jupyterlab/jupyterlab/issues/9399](https://github.com/jupyterlab/jupyterlab/issues/9399).
+
+### Evaluation report
+
+User reports on JupyterLab's accessibility are available at:[the jupyterlab/jupyterlab label `tag:accessibility`](https://github.com/jupyterlab/jupyterlab/labels/tag%3AAccessibility).
+
+### Formal complaints
+
+* [Write an issue on jupyter/accessibility](https://github.com/jupyter/accessibility/issues/new)
+* [Write an issue on jupyterlab/jupyterlab](https://github.com/jupyterlab/jupyterlab/labels/tag%3AAccessibility) and request it be labeled tag:accessibility.
```

Comment:
As an end user, how should I know which repo to submit my issue to?

Reply:
I don't know, because I don't know what the difference would be in opening it on one repo or the other.

My estimate is that

Writing on jupyterlab/jupyterlab
- Would likely be the place where the change would need to be made (or in a related repo like Lumino)
- Would be the place where we may be able to get support from people not as focused on accessibility but knowledgeable about JupyterLab
- Has lots and lots of issues and could risk burying new issues in short amounts of time
- To be honest, I don't follow all JupyterLab issues by default because there's too much activity

Writing on jupyter/accessibility
- Would give a lot of attention to an issue
- Would likely require opening another issue on the repo where the code changes need to be made (at some point)
- Has less people actively following and may not have the fastest support

Does this make you feel more confident if we should advise only one? Or how we should tell people to choose?

Reply:
So I do think we should advise only one, to keep things simple. But I too am not entirely sure which one we should use.

I think for me, a big part of it is about where the accessibility statement lives. If it's in the jupyterlab repo, then I think we should point them to create issues on the jupyterlab repo (with accessibility tag, just like you're doing). If the statement lives in the accessibility repo, then they should create issues on the accessibility repo. I dunno. I don't feel too strongly about that. 

But I do think it's pretty confusing to give somebody two options without even a hint about why they should choose one over the other. 

I think our own lack of razor sharp clarity on this bleeds through to the accessibility statement. What I mean is, we don't seem to have ourselves a clear criteria for when to submit to one or the other. A clear criteria, for example, would be something like: for all accessibility issues pertaining to JupyterLab create an issue on the JupyterLab repo; for all other Jupyter projects (JupyterHub, Lumino, etc.) submit an issue to the accessibility repo.

Reply:
I thought we had decided to keep the statement in this repo and x-reference to it from the Jupyterlab readme/docs?

Suggestion - even if these sections have been merged

```
* For accessibility-related matters within the broader Jupyter ecosystem or directly related with this statement - [Write an issue on jupyter/accessibility](https://github.com/jupyter/accessibility/issues/new)
* For accessibility issues related to JupyterLab only -[Write an issue on jupyterlab/jupyterlab](https://github.com/jupyterlab/jupyterlab/labels/tag%3AAccessibility) and request it be labeled tag:accessibility.
```